packages/e2e/features/ui/components/liveness/disable-start-screen.feature

```diff
@@ -24,6 +24,3 @@ Feature: Disable Start Screen
       Then I click the "FaceMovementAndLightChallenge" selectfield and select the "FaceMovementChallenge" option
       Then I see "FaceMovementChallenge"
       Then I see "liveness-detector" element
-      Then I see the "Face didn't fit inside oval in time limit." timeout error
```

Comment:
Why do these need to be removed?

Reply:
Rekognition updated the values for face distance which make it so these tests no longer work, only on the no light stuff, figure that's unrelated to this PR but just want to unblock this

Reply:
Oh gotcha, what updates would we need to get them passing? But make sense as a follow up

Reply:
We would need to update the video we use in the e2e test haha